astro/src/content/blog/i-just-want-authentication-to-work.mdx

```diff
@@ -1,35 +1,36 @@
 ---
 publish_date: 2023-11-17
 title: I Just Want Authentication To Work
-description: When I am writing an Application the last thing that I care about is setting up Authentication, I just want it to work!
+description: When I am writing an application the last thing that I care about is setting up authentication, I just want it to work!
 authors: Alex Patterson
 image: /img/blogs/i-just-want-authentication-to-work/i-just-want-auth-to-work.png
-categories: Company, News
-tags: award, security mag
+categories: Community
+tags: community story
```

Comment:
We typically have community stories tagged when a member of the community is interviewed. See https://fusionauth.io/blog/treefort-uses-fusionauth-for-all-auth for example.

Here are tags that I'd use:

authentication, application development, napster, music, 1990s.

Reply:
@mooreds should I do "Off Topic" for category?

Reply:
Maybe "Education"? We don't have Off Topic for a category.

Here's the list it is checked against: https://github.com/FusionAuth/fusionauth-site/blob/master/.github/known-blog-categories.txt